### ISSUE TITLE
A question about the patch loss

### ISSUE THREAD

**Issue by tydusky** [shangtse/robust-physical-attack#4]
> self.patch_loss_ = tf.nn.l2_loss(RED_MASK*(self.clipped_patch_ - np.tile(np.array([ 1.0, -0.9, -1]), (psize, psize, 1))))

Are the three numbers [1.0, -.9, -1] have special meanings?

**Comment by shangtse** [comment]
That just represents a red color. This loss encourages the resulting patch to be close to this red color.

**Comment by tydusky** [comment]
> That just represents a red color. This loss encourages the resulting patch to be close to this red color.

Thanks. Got it.
